scripts/templates/helper.py

```diff
@@ -895,8 +895,8 @@ def get_class_function_objs_exp(specs, cname):
                     exp_objects.append(obj)
                 else:
                     objects.append(obj)
-    objects = sorted(objects, key=lambda obj: (float(obj.get('version',"1.0"))*10000) + int(obj.get('ordinal',"100")))
-    exp_objects = sorted(exp_objects, key=lambda obj: (float(obj.get('version',"1.0"))*10000) + int(obj.get('ordinal',"100")))              
+    objects = sorted(objects, key=lambda obj: (float(obj.get('version',"1.0").translate({ord('.'):None}))*10000) + int(obj.get('ordinal',"100")))
+    exp_objects = sorted(exp_objects, key=lambda obj: (float(obj.get('version',"1.0").translate({ord('.'):None}))*10000) + int(obj.get('ordinal',"100")))              
```

Comment:
this would translate 2.0 -> 20 which would be less than 1.10 -> 110 but greater than 1.1 -> 11

Reply:
If at all possible, I highly recommend avoiding anything to do with the Ordinal, it is a mess waiting to happen...